cas-server-core/src/main/java/org/jasig/cas/services/RefuseRegisteredServiceProxyPolicy.java

```diff
@@ -55,7 +55,7 @@ public boolean equals(final Object o) {
             return false;
         }
 
-        return true;
+        return o instanceof RefuseRegisteredServiceProxyPolicy;
```

Comment:
I guess this test could be "smartly" merged with the test above: `if (!(o instanceof RegisteredServiceProxyPolicy)) {`